packages/x-data-grid/src/components/virtualization/GridVirtualScrollbar.tsx

```diff
@@ -135,11 +135,12 @@ const GridVirtualScrollbar = React.forwardRef<HTMLDivElement, GridVirtualScrollb
     useOnMount(() => {
       const scroller = apiRef.current.virtualScrollerRef.current!;
       const scrollbar = scrollbarRef.current!;
-      scroller.addEventListener('scroll', onScrollerScroll, { capture: true });
-      scrollbar.addEventListener('scroll', onScrollbarScroll, { capture: true });
+      const eventOptions: AddEventListenerOptions = { capture: true, passive: true };
```

Comment:
lint: we can skip the type annotation I think, it keeps thing more readable to let inference handle them. And the name could be shortened to `options`, it's pretty clear what's going on here.